### ISSUE TITLE
Midline liver

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7346]
**Preferred term label:**
Midline liver

**Synonyms**

**Definition (free text, please give PubMed ID)**
Position of the liver across the middle of the body instead of being in its normal position to the right of the stomach.
PMID:12432114
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:613751 
PMID: 9916847  2/3
**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
